package/src/i18n/hi.json

```diff
@@ -40,6 +40,7 @@
   "Photos": "तस्वीरें",
   "Photos and Videos": "तस्वीरें और वीडियों",
   "Pin to Conversation": "बातचीत में पिन करें",
+  "Pinned by": "द्वारा पिन किया गया",
```

Comment:
@khushal87 can you please validate?

Reply:
This isn't correct Hindi, but we cannot do any thing. The name come first and followed by text but we have this text followed by name in Engish.